src/concrete/ml/quantization/post_training.py

```diff
@@ -478,6 +515,17 @@ def _quantize_layers(self, *input_calibration_data: numpy.ndarray):
 
         constants: Set[str] = set(self.quant_params.keys())
 
+        # Check if the model has only GLWE supported linear layers.
+        # In this case, use analytical calibration which is much faster
+        fast_calibration = True
+        for node in graph.node:
```

Comment:
that's adding another full traversal of the onnx graph here. We should probably exit the loop when we encounter an op that's not supported

Reply:
yes, you're right. traversing the graph in this way is pretty fast though, I'm guessing the graph is in memory and we're just processing some pointers.